popup/popup.js

```diff
@@ -527,17 +536,9 @@ Object.assign(handleEvent, {
   },
 
   openLink(event) {
-    if (!chrome.windows || !prefs.get('openEditInWindow', false)) {
-      handleEvent.openURLandHide.call(this, event);
-      return;
-    }
     event.preventDefault();
-    chrome.windows.create(
-      Object.assign({
-        url: this.href
-      }, prefs.get('windowPosition', {}))
-    );
-    close();
+    API.openURL({url: this.href});
+    if (!(FIREFOX && prefs.get('openEditInWindow'))) window.close();
```

Comment:
We have some similar logic in `openURL`. Can we merge them?

Reply:
The link is opened through the API. `window.close()` was already there, but tabs in different FF windows can be ignored via the pref to avoid the focus bug, instead of needing to check. Could it be moved? I suppose, but it's not necessary.